packages/rs-drive-abci/tests/strategy_tests/main.rs

```diff
@@ -1421,7 +1421,7 @@ mod tests {
                     .unwrap()
                     .unwrap()
             ),
-            "839048d5a1b50ad80658f02aee27a20e9a05d2eecfde9170ffe2242bd5bfd20f".to_string()
+            "a8d44b2db75668043189416351db1d812e58dccda3c4a35d319299754bdebd27".to_string()
```

Comment:
The hash had to be updated because in GroveDB's PR an unnecessary operation that affected costs was removed